### DISCUSSION TITLE
Need to synchronize Desktop folder

### DISCUSSION THREAD

**Discussion by juan-cabrera** [abraunegg/onedrive#2800] [Q&A] [answered]
Hi,

I have a /home/user/OneDrive folder where things are synchronized. I tried a bind mount

mkdir /home/user/OneDrive/Desktop_Linux
sudo mount --bind /home/user/Desktop  /home/user/OneDrive/Desktop_Linux

runing ondrive --syncronise does not see the Desktop_Linux folder.

Is there a way to synchronize a folder outside the OneDrive one?

**Comment by abraunegg** [comment, marked as answer]
@juan-cabrera 
> Is there a way to synchronize a folder outside the OneDrive one?

Use a symbolic link to your data

**Comment by juan-cabrera** [comment]
Thanks this works. I realized that skip_symlinks was  "true" in my config file
